Sources/CodeMirror-SwiftUI/Representables/CodeView.swift

```diff
@@ -124,8 +124,10 @@ extension CodeView {
     let webView = WKWebView(frame: .zero, configuration: configuration)
     webView.navigationDelegate = context.coordinator
     #if os(OSX)
-    webView.setValue(false, forKey: "drawsBackground")
+    webView.setValue(false, forKey: "drawsTransparentBackground")
```

Comment:
This is causing a white flick on macOS 10.15.6 for me. Can you check on your side please?

Reply:
> This is causing a white flick on macOS 10.15.6 for me. Can you check on your side please?

Actually, I can not see any difference, but it seems `"drawsBackground"` is better.
https://developer.apple.com/forums/thread/121139?answerId=376655022#376655022